README.md

```diff
@@ -77,7 +77,7 @@ pip install -r requirements.txt
       -cp COMBAT_POWER, --combat-power COMBAT_POWER Transfer Pokemon that have CP less than this value (default 100)",
       -iv IV, --pokemon-potential IV                Set the ratio for the IV values to transfer (DEFAULT 0.4 eg. 0.4 will transfer a pokemon with IV 0.3)
       -ri, --recycle-items                          Recycle unneeded items automatically
-      -if LIST, --item-filter LIST                  Pass a list of unwanted items to recycle when collected at a Pokestop (e.g, [\"101\",\"102\",\"103\",\"104\"] to recycle potions when collected). Requires --recycle-items. 
+       --item-filter DICT                            Pass a dict of pairs (item: quantity) to recycle when collected at a Pokestop. (e.g {"1": { "keep": 150} will keep up to 150 Pokeballs in the inventory). Requires --recycle-items. Only works in the config file.
```

Comment:
remove item filter entirely from the README or is the change I did good enough ?